test/rstar.jl

```diff
@@ -25,8 +25,8 @@ end
     @testset "samples input type: $wrapper" for wrapper in [Vector, Array, Tables.table]
         # In practice, probably you want to use EvoTreeClassifier with early stopping
         classifiers = (
-            EvoTreeClassifier(; nrounds=1_000, eta=0.1),
-            Pipeline(EvoTreeClassifier(; nrounds=1_000, eta=0.1); operation=predict_mode),
+            EvoTreeClassifier(; nrounds=1_000, eta=0.01),
```

Comment:
Is it clear from the changelog why the parameters have to be changed? Or was the previous choice non-standard?

Reply:
Not from the changelog, but the failures seem to come from https://github.com/Evovest/EvoTrees.jl/pull/250 changing defaults from `(max_depth = 5, nbins=32)` to `(max_depth=6, nbins=64)`. Reverting either of these to the former default causes the tests to pass again.

Looks like a lot of parameter combinations cause the tests to pass, but just barely. Perhaps a better approach is to increase the rtol.